### PR TITLE
MAILET-149 Introduce ICS mailets workflow tests 

### PR DESCRIPTION
**Pull request by aduprat**

### CODE REVIEW

File: server/mailet/integration-testing/src/test/java/org/apache/james/transport/mailets/ICSAttachmentWorkflowTest.java [hunk 1]
```diff
@@ -417,7 +416,6 @@ public void headersShouldBeAddedInMailWhenOneICalAttachment() throws Exception {
         }
     }
 
-    @Ignore("See JIRA issue MAILET-151")
```
Comment (rouazana) [inline server/mailet/integration-testing/src/test/java/org/apache/james/transport/mailets/ICSAttachmentWorkflowTest.java]: this commit should be a fixup, there is no reason to fix it only now I guess

**Comment by chibenwa** [comment]
You miss commits from here, don't you : https://github.com/linagora/james-project/pull/573
You also might need some more fixups mentionned on there.

However for last commit, could you please add a little test in StripAttachmentTest (Unit tests) ?

**Comment by aduprat** [comment]
Fixup from #573 cherry-picked
New test added about empty filename.

**Comment by chibenwa** [comment]
merged

**Pull request merged**